akka-typed-testkit/src/main/scala/akka/typed/testkit/Effects.scala

```diff
@@ -20,7 +20,7 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
 abstract class Effect
 
 object Effect {
-  @SerialVersionUID(1L) final case class Spawned(childName: String) extends Effect
+  @SerialVersionUID(1L) final case class Spawned(childName: String, props: Option[Props]) extends Effect
```

Comment:
I kind of like having individual `Effects` for each spawn type. I can make them all extend some abstract class named `SpawnedEffect` so it could still be pattern match like:
```
case spawned: SpawnedEffect => // I don't care which type of Spawning it was.
```

WDTY?